packages/node/src/workers/cloud-platforms/aws.ts

```diff
@@ -1,4 +1,5 @@
 import AWS from 'aws-sdk';
+import JSON from 'json-bigint';
```

Comment:
I would prefer to not override the default global `JSON`. Can we rather call this something else? e.g. 
```suggestion
import JSONBigInt from 'json-bigint';
```